apiserver/modelmanager/modelmanager.go

```diff
@@ -537,7 +536,14 @@ func (m *ModelManagerAPI) ModelInfo(args params.Entities) (params.ModelInfoResul
 	return results, nil
 }
 
+type pluggableAuthFunc func(names.UserTag) error
+
 func (m *ModelManagerAPI) getModelInfo(tag names.ModelTag) (params.ModelInfo, error) {
+	return m.getModelInfoPluggableAuth(tag, m.authCheck)
+}
+
+func (m *ModelManagerAPI) getModelInfoPluggableAuth(tag names.ModelTag,
```

Comment:
I don't see what this pluggableAuth change is for. The only functional difference I can see is that we are passing in something that always return nil when called by CreateModel. Why isn't the current authCheck sufficient in that case?